package.json

```diff
@@ -43,9 +43,9 @@
     "@types/prettier": "^2.0.0",
     "@types/reflect-metadata": "^0.1.0",
     "@types/rimraf": "^3.0.0",
-    "class-transformer": "^0.3.1",
+    "class-transformer": "^0.4.0",
     "class-validator": "^0.12.2",
-    "class-validator-jsonschema": "^2.0.0-rc1",
+    "class-validator-jsonschema": "^2.2.0",
```

Comment:
Note that `class-validator-jsonschema` isn't compatible with `class-transformer >0.3.1` atm: https://github.com/epiphone/class-validator-jsonschema/issues/55

Reply:
reverted to the 2.0.0-rc1